November 2025 Newsletter blog (#464)
ddev/ddev.com: src/content/blog/ddev-november-2025-newsletter.md

## What's New

- **Claude Code AI-assisted PRs for DDEV Training** → Learn how to use Claude Code to create pull requests for DDEV [Read more↗](claude-code-ai-pr-for-ddev-contributor-training.md)

## Community Highlights

- **A Love Letter to DDEV**: Ryan Stubbs shares his appreciation for DDEV and its impact on development workflows [Read more↗](https://ryanstubbs.co.uk/2025/11/a-love-letter-to-ddev/)
- **Claude Code Plugins for Drupal/DDEV**: Community member Luděk Kvapil has created custom Claude Code commands and plugins for DDEV and Drupal development workflows [View on GitHub↗](https://github.com/siva01c/claude-plugins)
- **Profiling Your Drupal App with New Relic and DDEV**: Practical guide session from DrupalCon Vienna 2025 [Session details↗](https://events.drupal.org/vienna2025/session/profiling-your-drupal-app-new-relic-ddev-practical-guide) • [Watch on YouTube↗](https://www.youtube.com/watch?v=S5oPq1FqB1I&list=PLpeDXSh4nHjR5zd_mBGhzW4YBwuv5zs3d&index=113)

## Community Video Tutorials

> - **Drupal CMS und Core für Projektentwicklung unter Windows 11 von Null installieren (German)**: Complete guide to installing Drupal CMS and Core for project development on Windows 11 from scratch [Watch on YouTube↗](https://www.youtube.com/watch?v=AwORq6A07pc)

Reviewer: ```suggestion
- **Drupal CMS und Core für Projektentwicklung unter Windows 11 von Null installieren (German)**: Complete guide to installing Drupal CMS and Drupal Core for project development on Windows 11 from scratch [Watch on YouTube↗](https://www.youtube.com/watch?v=AwORq6A07pc)
```

> - **Drupal 11 für Entwicklerinnen und Sitebuilderinnen (German)**: Udemy course for Drupal 11 developers and site builders [Enroll with coupon↗](https://www.udemy.com/course/drupal-11-fur-entwicklerinnen-und-sitebuilderinnen/?couponCode=MT251103G2)

Reviewer: ```suggestion
- **Drupal 11 für Entwickler:innen und Sitebuilder:innen (German)**: Udemy course for Drupal 11 developers and site builders [Enroll with coupon↗](https://www.udemy.com/course/drupal-11-fur-entwicklerinnen-und-sitebuilderinnen/?couponCode=MT251103G2)
```
the colons are required in the title. without you are only addressing female developers and site builder, by using the colon you are using the gender inclusive form. strictly speaking the prefered special character instead of the colon is the star (*) but the colon is fine as well.

  [Add to Google Calendar](https://calendar.google.com/calendar/render?action=TEMPLATE&text=Git%20bisect%20for%20fun%20and%20profit&dates=20260226T150000Z/20260226T160000Z&details=Join%20the%20DDEV%20training%20session%20via%20Zoom.%0ALink%3A%20https%3A%2F%2Fus02web.zoom.us%2Fj%2F7315692237%3Fpwd%3DRHR6NUkwb0g5WXIzS2NOcXRucCthZz09%0AMeeting%20ID%3A%20731%20569%202237%0APasscode%3A%2012345&location=Online&trp=true) •
  [Download .ics](/files/ics/ddev-2026-02-26.ics)

- **March 26, 2026 at 10:00 US ET / 15:00 CET — Using `git worktree` with DDEV projects and with DDEV itself**
  [Add to Google Calendar](https://calendar.google.com/calendar/render?action=TEMPLATE&text=Using%20git%20worktree%20with%20DDEV%20projects%20and%20with%20DDEV%20itself&dates=20260326T140000Z/20260326T150000Z&details=Join%20the%20DDEV%20training%20session%20via%20Zoom.%0ALink%3A%20https%3A%2F%2Fus02web.zoom.us%2Fj%2F7315692237%3Fpwd%3DRHR6NUkwb0g5WXIzS2NOcXRucCthZz09%0AMeeting%20ID%3A%20731%20569%202237%0APasscode%3A%2012345&location=Online&trp=true) •
  [Download .ics](/files/ics/ddev-2026-03-26.ics)

- **April 23, 2026 at 10:00 US ET / 16:00 CEST — Creating, maintaining and testing add-ons**

>   Re‑run of our popular add‑on training. [Previous session recording↗](https://www.youtube.com/watch?v=TmXqQe48iqE)

Reviewer 2: ```suggestion
  2026-updated version of our popular add‑on training. [Previous session recording↗](https://www.youtube.com/watch?v=TmXqQe48iqE)
```
